src/tracer/mod.rs

```diff
@@ -111,6 +123,18 @@ impl Tracer {
             .unwrap()
             .clone()
     }
+
+    pub fn count(&mut self) {
```

Comment:
maybe inc_trace_size()? since inc means add one. count does not imply the number.